joomla/joomla-cms
# [4.0] Change collations of com_finder tables to unicode on MySQL databases also in the component's install sql

Pull Request for Issue [https://github.com/joomla/joomla-cms/pull/28425#issuecomment-603713289](https://github.com/joomla/joomla-cms/pull/28425#issuecomment-603713289).

### Summary of Changes

Same as in PR #28425 but for the component-specific install sql.

This is not really used anymore but still shall be kept up-to-date according to @wilsonge 's comment here: [https://github.com/joomla/joomla-cms/pull/28425#issuecomment-603745707](https://github.com/joomla/joomla-cms/pull/28425#issuecomment-603745707).

For PostgreSQL only comments and code style have changed so it is more equal to the corresponding section in the joomla.sql file.

### Testing Instructions

Code review:
- Compare the mysql/joomla.sql and the install.mysql.sql files and verify that beside comments and empty lines, the sections for creating com_finder tables in both files are the same.
- Compare the postgresql/joomla.sql and the install.postgresql.sql files and verify that beside comments and empty lines, the sections for creating com_finder tables in both files are the same.
- Verify that the changes in the install.XXX.sql files are basically the same as done in PR #28425 in the corresponding joomla.sql file.

### Expected result

Files install.mysql.sql and install.postgresql.sql of com_finder are up-to-date with the changes in PR #28425 for joomla.sql.

### Actual result

Files install.mysql.sql and install.postgresql.sql of com_finder are **not** up-to-date with the changes in PR #28425 for joomla.sql.

### Documentation Changes Required

None.
